### ISSUE TITLE
Bookmarks in the root of "Other bookmarks" can't be moved

### ISSUE THREAD

**Issue by rpkamp** [rpkamp/chrome-minimal-bookmarks-tree#101]

**Comment by oleedd** [comment]
It is a dragula problem. It requires a parent element. So `<ul id="bookmarks">` should be used in this case.

**Comment by rpkamp** [comment]
Yes, the parent of the bookmark elements should have some data attached to it that indicates the folder ID, otherwise it doesn't know where the bookmarks should be moved.

Probably easiest to solve this together with https://github.com/rpkamp/chrome-minimal-bookmarks-tree/issues/125, since that will change how folders are rendered anyway.
